tests/testutils/kubeutils/kind_cluster.go

```diff
@@ -96,9 +101,16 @@ func (k *KindCluster) Create() {
 	require.NoError(k.Testcase, err)
 	k.Clientset, err = kubernetes.NewForConfig(restConfig)
 	require.NoError(k.Testcase, err)
+
+	for _, pod := range []string{
+		"kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd-cluster", "coredns", "kindnet", "kube-proxy",
+	} {
+		k.WaitForPods(fmt.Sprintf("%s-.*", pod), "kube-system", 2*time.Minute)
+	}
+	k.WaitForPods("local-path-provisioner-.*", "local-path-storage", 2*time.Minute)
 }
 
-func (k *KindCluster) Delete() {
+func (k *KindCluster) Teardown() {
```

Comment:
This change is fine, but can it be split to its own PR so it can reduce the diff of this PR?